example/hyperparam/train_keras.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import print_function
+
+import warnings
+
+import math
+import numpy as np
+import pandas as pd
+
+import click
+from sklearn.metrics import mean_squared_error, mean_absolute_error, r2_score
+from sklearn.model_selection import train_test_split
+
+from keras.models import Sequential
+from keras.layers import Dense, Lambda
+from keras.optimizers import Adamax
+
+import mlflow
+import mlflow.keras
+
+
+def eval_metrics(actual, pred):
+    rmse = np.sqrt(mean_squared_error(actual, pred))
+    mae = mean_absolute_error(actual, pred)
+    r2 = r2_score(actual, pred)
+    return rmse, mae, r2
+
+
+def get_standardize_f(train):
+    mu = np.mean(train, axis=0)
+    std = np.std(train, axis=0)
+    return lambda x: (x - mu) / std
+
+
+from keras.callbacks import Callback
+
+
+class MLflowLogger(Callback):
+    def __init__(self):
+        super(MLflowLogger, self).__init__()
+        self._score_f = None
+
+    def on_epoch_end(self, epoch, logs=None):
+        # log standard keras metrics
+        if logs:
+            for k, v in logs.items():
+                mlflow.log_metric(k, v)
+
+
+class MLflowCheckpoint(Callback):
```

Comment:
Add a doc comment for this; it's cool but we should tell people why we added this